src/st_keyup/frontend/main.js

```diff
@@ -45,9 +45,11 @@ function onRender(event) {
 
     if (type == "password") {
       input.type = "password"
+    }else if(type == "text"){
+      input.type = "text"
     }
     else {
-      input.type = "text"
+      input.type = type
```

Comment:
Are there particular other types you would like to support? I certainly don't want people passing type=radio, as that wouldn't make too much sense.

Reply:
Good point, the other input types I wanted to add were: "tel", "number" & "email".

Reply:
I've updated the doc string, and changed the css to support darkmode 😊